plugin/validator.go

```diff
@@ -22,9 +22,17 @@ func NewValidator(opts ValidatorOpts) (*Validator, error) {
 	return &Validator{vm: vm}, nil
 }
 
+func (v *Validator) ValidateSchema(schema string) (string, error) {
+	return v.vm.CallByParams("validate_plugin_schema", schema)
+}
+
 func (v *Validator) LoadSchema(schema string) (string, error) {
-	pluginName, err := v.vm.CallByParams("load_plugin_schema", schema)
-	return pluginName, err
+	return v.vm.CallByParams("load_plugin_schema", schema)
+}
+
+func (v *Validator) UnloadSchema(schema string) error {
```

Comment:
What would happen if a caller is unloading schema while that schema is being used? Could this result in any unwanted scenarios?

Reply:
We may want to add a guard against this, but the same issue exists in the load `LoadSchema` call as well since it also updates the `Plugins` instance. It might make more sense to move that logic out of this module and into the application that is using it; WDYT?

Reply:
I agree, this is definitely an issue of how it is used vs. what this library can do.